### DISCUSSION TITLE
Share Database / Libraries between different Docker Environments

### DISCUSSION THREAD

**Discussion by pabera** [immich-app/immich#14470] [Q&A] [answered]
I started to use Immich on my Synology NAS using Docker with success. It runs great and I want to use it. My existing photo/video repository is huge and the NAS collapses running all the background jobs. I tried pausing them but ETAs for completion are long. Also, when using the CPU, many other services on the NAS struggle to continue.

I though of outsourcing the initial process of re-indexing the photo/video repository to my Windows PC. It has much more CPU power, supports GPU etc.

I updated the `.env` and `docker-compose.yml` to refer to the previously created DB and upload paths (through normal Windows network share).

Immich starts properly on Windows but when navigating to the UI, I am being asked to create an administrator account, which leaves me thinking it does not actually refer to my network shared, where my Postgres DB is stored.

1. Do you think this can work?
2. What are ways to check whether it used the right path or not.

**Comment by bo0tzz** [comment, marked as answer]
You absolutely should never point multiple postgres containers at the same folder. If you're lucky it will simply not work, but more likely things will break horribly.

What you're looking for is https://immich.app/docs/guides/scaling-immich. Another option is to start your instance from scratch on the more powerful system, then use https://immich.app/docs/administration/backup-and-restore to move it after the processing is done.

**Reply by pabera** [reply]
I was totally aware of the risk. Since I am not yet using it in production, this was ok for me. But it's good to know of course.

The guides you provided are interesting and sound like good opportunities.
Thanks for the swift reply

**Comment by mmomjian** [comment]
Also you should never use a network share for the database
